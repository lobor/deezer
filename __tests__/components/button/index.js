import React from 'react';
import Button from 'components/button';
import renderer from 'react-test-renderer';

describe('test component button', () => {
  it('test render with text', () => {
    const component = renderer.create(
      <Button>Toto</Button>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })

  it('test render primary', () => {
    const component = renderer.create(
      <Button primary />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
