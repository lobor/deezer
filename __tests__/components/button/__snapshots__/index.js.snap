// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component button test render primary 1`] = `
<button
  className="btn btn-primary"
/>
`;

exports[`test component button test render with text 1`] = `
<button
  className="btn"
>
  Toto
</button>
`;
